setup.py

```diff
@@ -55,7 +55,7 @@ def to_uri(filename):
 
 setup(
     name='ismrmrd',
-    version='1.10.0',
+    version='1.11.0',
     author='ISMRMRD Developers',
     author_email='dchansen@gradientsoftware.net',
```

Comment:
We should maybe update or remove this email address since I am not sure if it is being monitored.

Reply:
We should absolutely change that email address - it never existed.